assets/app/views/pods.html

```diff
@@ -8,10 +8,58 @@
           <em>{{emptyMessage}}</em>
         </div>
       </div>      
-      <div style="margin-bottom: 10px;" ng-repeat="pod in pods">
-        <h3>{{pod.metadata.name}} ({{pod.status.phase}})</h3>
+      <div style="margin-bottom: 10px;" ng-repeat="pod in pods | orderObjectsByDate : true" class="tile">
+        <h2>{{pod.metadata.name}}</h2>
+        <div>
+          <span style="margin-right: 5px;">Status:  {{pod.status.phase}}</span>
+          <span ng-switch="pod.status.phase">
+            <span ng-switch-when="Succeeded" class="fa fa-check text-success" aria-hidden="true" style="margin-right: 5px;"></span>
+            <span ng-switch-when="Terminated" class="fa fa-times text-danger" aria-hidden="true" style="margin-right: 5px;"></span>
+            <span ng-switch-when="Pending" class="fa fa-refresh fa-spin" aria-hidden="true" style="margin-right: 5px;"></span>
+            <span ng-switch-default class="fa fa-refresh text-success" aria-hidden="true" style="margin-right: 5px;"></span>
```

Comment:
Currently the only other phase is "Running"  i can make that be running explicitly and pick something generic in case they add states later